components/dashboard/src/components/MonacoEditor.tsx

```diff
@@ -34,6 +34,10 @@ export default function MonacoEditor(props: { classes: string, disabled?: boolea
     }
   }, [ props.value ]);
 
+  useEffect(() => {
+    monaco.editor.setTheme(document.documentElement.classList.contains('dark') ? 'vs-dark' : 'vs');
```

Comment:
question: would it make sense to move it before L16? is it guaranteed then that the theme is already set before the instance is created?

Reply:
This is independent of the instance (`monaco.editor` vs `editorRef.current`) and acts like a global I think (thus not tied to any particular instance).

Reply:
true. the thing is, if it's already constructed with one, and you change it immediately afterwards, would it render with the "wrong" theme first?

Reply:
When testing with light and dark themes, it always looked correct / never flashed a wrongly-colored render.